lib/src/rules/directives_ordering.dart

```diff
@@ -83,10 +124,24 @@ class _Visitor extends SimpleAstVisitor {
 
   @override
   void visitCompilationUnit(CompilationUnit node) {
+    _checkDartImportGoFirst(node);
+    _checkPackageImportBeforeRelative(node);
+  }
```

Comment:
We have the messages all together in one block, so we think it will be fine, the user will be able to fix it with just one command, the only problem, is when we implement the rest of the parts, one line can have more than one lint, and in that way, for the next parts we want to implement it with a Set, the idea is with the guideline priority we won't mark a line more than once